app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarLayoutViewModel.kt

```diff
@@ -134,6 +134,7 @@ class OmnibarLayoutViewModel @Inject constructor(
         val showTabsMenu: Boolean = true,
         val showFireIcon: Boolean = true,
         val showBrowserMenu: Boolean = true,
+        val showChatMenu: Boolean = true,
```

Comment:
should this be enabled by default?

Reply:
I tried setting this to false but it messes up the animation, it needs to be there by default. 
I’ll address the issue along with that other reported bug separately.